### PR TITLE
Remove light client commands and configuration

### PR DESCRIPTION
**Pull request by romac**
See: #673

## Description

Remove the light client configuration from the global configuration and remove the `light add` and `light rm` commands.

The peer id for the light client is now fetched at light client initialization time from the RPC address configured for the chain.

______

For contributor use:

- [x] Updated the __Unreleased__ section of [CHANGELOG.md](https://github.com/informalsystems/ibc-rs/blob/master/CHANGELOG.md) with the issue.
- [ ] If applicable: Unit tests written, added test to CI.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Updated relevant documentation (`docs/`) and code comments.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: config_example.toml [hunk 1]
```diff
@@ -26,6 +26,7 @@ id = 'ibc-1'
 rpc_addr = 'tcp://localhost:26557'
 grpc_addr = 'tcp://localhost:9091'
```
Comment (ancazamfir) [inline config_example.toml:27]: change to `http://...`

File: scripts/init-clients [hunk 2]
```diff
@@ -68,33 +68,6 @@ CHAIN_2_RPC_ADDR="http://127.0.0.1:$CHAIN_2_RPC_PORT"
 echo "Building the Rust relayer..."
 cargo build -q
 
-# cleanup the client entries from config
```
[the remark this answers is not in the document]
Comment (ancazamfir) [reply scripts/init-clients:71]: And maybe change the name of the script from `init-clients` to something like `init-hermes`?

**Review by andynog — APPROVED** [review]
The changes looks good to me. The e2e test changes look good to me too and all the new configs seem to be fine. I'm assuming the docs updates will be handled on another issue so it's all good. Good job. Thanks !

**Review by ancazamfir — APPROVED** [review]
Only a couple of minor changes, otherwise it looks great! Thanks Romain!

**Comment by ancazamfir** [comment]
FYI, I did some tests with `cosmoshub-4` and even with `30sec` timeouts i see errors on client update once in a while. We timeout in runtime's `build_header()` (coming many times from tendermint-rs light client verify).
There are other timeouts also on `init_event_monitor()`. It may be the full node I am using that it's busy, not sure. I think these are not related to this PR but I will open an issue to investigate.

**Pull request merged**